.changeset/smart-grapes-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-labeled-field": patch
```

Comment:
Note: Although renaming a prop is technically a breaking change, I marked this as a patch change since the LabeledField component isn't released yet (it's in the feature/labeled-field branch so the previous prop hasn't been available for consumers. Let me know if there are any concerns with this!

Reply:
sounds good to me, thanks for the clarification!